js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+
+export default class extends baseVw {
+  constructor( options = { } ) {
+    super( options );
+
+    this.options = options;
+    this.settings = app.settings.clone( );
```

Comment:
To handle the appearance, you should get both the parameters from local settings, using:

```
app.localSettings.get('macStyleWinControls')
app.localSettings.get('visualEffect') //this doesn't exist yet
```

When the form is saved, you should set them both using localSettings. 

The pageNav view has a listener that will make the style changes when the local settings for macStyleWinControls is changed.

We'll add something similar later for visualEffects, the class to deactivate the visual effects doesn't exist yet.